Mathlib/MeasureTheory/Group/Measure.lean

```diff
@@ -761,6 +784,18 @@ theorem isHaarMeasure_map [BorelSpace G] [TopologicalGroup G] {H : Type _} [Grou
 #align measure_theory.measure.is_haar_measure_map MeasureTheory.Measure.isHaarMeasure_map
 #align measure_theory.measure.is_add_haar_measure_map MeasureTheory.Measure.isAddHaarMeasure_map
 
+/-- The image of a Haar measure under right multiplication is again
+a Haar measure. -/
+@[to_additive isHaarMeasure_radd]
+theorem isHaarMeasure_rmul [BorelSpace G] [TopologicalGroup G] [T2Space G] (g : G) :
+    IsHaarMeasure (Measure.map (· * g) μ) where
+  toIsMulLeftInvariant := isMulLeftInvariant_rmul g
```

Comment:
This one should generalize to something like
```suggestion
theorem isHaarMeasure_smul [BorelSpace G] [TopologicalGroup G] [T2Space G] (a : α) :
    IsHaarMeasure (Measure.map (a • · : G → G)  μ) where
  toIsMulLeftInvariant := isMulLeftInvariant_smul a
```
though you will have to add some assumption on `α`. You should then be able to prove `isHaarMeasure_rmul` in a single line like I did with `isMulLeftInvariant_rmul`